src/main/main.c

```diff
@@ -26,6 +26,8 @@ static	t_error_code	handle_input(t_shell_op *sp, t_read_input read_func)
 	if (SUCCESS != err)
 		return (err);
 	err = parser_check_tokens(sp);
+	if (0 == **(sp->input))
```

Comment:
I think this should be in line 23:
`
if (NULL == sp->input || NULL == *(sp->input) || 0 == **(sp->input))
		return (EOF_SUCCESS);
		`